commerce/domain/tax/service/classes/TaxServiceExtensionResolverSample.cls

```diff
@@ -0,0 +1,36 @@
+// This sample is for the situation when different tax behaviors need to be implemented for different locales.
+// These tax behaviors can be -
```

Comment:
Please add a basic explanation of what markets is and a link to public documentation. 

Second, the idea of a resolution state and extension provider should not be introduced in comments.  Please coordinate with your doc writer to have some public documentation outside of this repo.